common/assets/scss/core/_elements.scss

```diff
@@ -5,3 +5,8 @@ abbr {
 abbr[title] {
   cursor: help;
 }
+
+img {
+  display: inline-block;
+  width: 100%;
```

Comment:
a) Should this apply to all images?

b) Is this what we actually want? 

In mobile view the image will fill the viewport

![Move_details_for_AARDVARKY__ANDREW](https://user-images.githubusercontent.com/4856/80340195-1b9d1380-8858-11ea-94ff-bbea87f5b8f6.png)

vs shrinking in the desktop view

![Move_details_for_AARDVARKY__ANDREW](https://user-images.githubusercontent.com/4856/80340374-6d459e00-8858-11ea-983e-a95f61da5721.png)

Might `max-width` be more appropriate here?

Reply:
Good spot. This was a bit of a late night one...

I think we probably do want it to fill the container on smaller layouts. 

Also, the shrinkage on larger screens is a bit of a hack using the GOV.UK grid...probably another one to revisit at some point.